contributors/devel/api-conventions.md

```diff
@@ -1240,6 +1240,8 @@ latter if the label/annotation is specific to `foo`
 the resource doesn't need to know about, experimental fields that aren't
 intended to be generally used API fields, etc. Beware that annotations aren't
 automatically handled by the API conversion machinery.
+  - The key name of [taint](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) should also follow above naming conversion.
```

Comment:
Line 1231 already enumerates this guidance as applying to "labels, annotations and other generic map keys". You could probably remove this line and just change line 1231 to include "taints".

Reply:
Good point, fixing